reconcile/utils/oc.py

```diff
@@ -472,6 +473,22 @@ def get_version(self):
         cmd = ['version', '--request-timeout=5']
         return self._run(cmd)
 
+    @retry(exceptions=InconsistentProjectCountError)
+    def get_project_names(self, consistent=False):
+        projects = self.get_projects()
+        if consistent:
+            projects_attempt_2 = self.get_projects()
```

Comment:
Don't we have a race condition here? A different piece of automation can add/remove projects that we're not allowed to modify (i.e, somebody else might be operating outside our limited `managedResourceNames`).